cPRONTO/oppni.py

```diff
@@ -1297,9 +1328,18 @@ def local_exec(script_path):
     # logger = logging.getLogger(script_path + '.log')
 
     # make it executable
+
+    #adlofts add jun 26 2018
+    script_path = list(script_path)
```

Comment:
explain why this must be done.

Reply:
I think this is redundant after the correction done at 1677, unless at jobs_list[idx_beg:idx_beg + num_procs] changes it to dictionary. jobs_list is indeed a list.